src/store/modules/store.js

```diff
@@ -5,30 +5,41 @@ export default {
         error: false,
         login_progress: false,
         register_progress: false,
+        add_poll: false,
+        delete_poll: false,
         user_exist: {},
         token_value: {},
+        polls: {},
+        search: {},
+        updatedpoll: {}
     },
     getters: {
```

Comment:
try using ...mapState() instead of getters if no computation is happening in getters!!